dummy-api/profile/overview.ts

```diff
@@ -0,0 +1,458 @@
+import PlayerTshirtIcon from '@/icons/PlayerTShirtPngIcon.png'
+import PlayerUpIcon from '@/icons/PlayerUpPngIcon.png'
+import SecondLeagueIcon from '@/icons/SecondLeague.png'
+import ThirdLeagueIcon from '@/icons/ThirdLeague.png'
+
+import { IProfileCareerData } from './career'
+import { IProfileMatchesSeason } from './matches'
+
+export interface IProfileOverviewData {
+  inShort: {
+    title: string
+    icon: string
+  }[]
+  transfers: {
+    id: number
+    from: {
+      logo: string
+      name: string
+    }
+    to: {
+      logo: string
+      name: string
+    }
+    type: string
+    date: string
+  }[]
+  similarPlayers: {
+    name: string
+    image: string
+    score: number
+    premium: boolean
+    age: number
+    scoreTrend: 'up' | 'down'
+    position: string
+    team: string
+    competition: string
+  }[]
+  experience: {
+    data: {
+      competetion: string
+      competitionLogoUrl: string
+      seasons: number
+      matches: number
+      goals: number
+      avgGoals: number
+      avgMinutes: number
+    }[]
+    total: {
+      seasons: number
+      matches: number
+      goals: number
+      avgGoals: number
+      avgMinutes: number
+    }
+  }
+  playerData: {
+    altPosition: string
+    position: string
+    betterLeg: string
+    height: number
+    weight: number
+    location: string
+    age: number
+    name: string
+    clubLogo: string
+    video: string
+  }
+  carrer: IProfileCareerData[]
+  lastMatches: {
+    playerStandedOut: boolean
+    data: IProfileMatchesSeason[]
+  }
```

Comment:
these are single panels + typo:), don't forget to adjust components
```suggestion
  career: IProfileCareerData
  lastMatch: {
    playerStandedOut: boolean
    data: IProfileMatchesSeason
  }
```